src/models/user.js

```diff
@@ -59,7 +59,7 @@ export default (sequelize, DataTypes) => {
       type: DataTypes.TEXT,
       validate: {
         is: {
-          args: /^[A-Za-z]+[A-Za-z0-9 _.,!?;"']+$/i,
+          args: /^[A-Z]+[A-Z0-9 _\.\-,!?;"']+$/i,
```

Comment:
" ? " is a special character in Regular Expression patterns. You will need to escape it with " \ "